custom_msg/custom_msg.py

```diff
@@ -39,6 +39,9 @@ async def msg_create(self, ctx: commands.GuildContext, specified_channel: Option
 
     @msg_cmd.command(name="edit")  # type: ignore
     async def msg_edit(self, ctx: commands.Context, message: discord.Message):
+        """Use message link when specifying what message to edit.
+        Message ID will likely not work."""

```

Comment:
The first line is made bold as a summary of the command in Red's help messages, so I'd recommend adding a line that would work better for that. It'd also be handy to provide some context as to why IDs can be problematic, else it sort of makes it look like an issue with the cog.

```suggestion
        """Edit a message sent by this cog
        It's best to use a message link when specifying which message to edit.
        Providing a message ID is likely to fail for older messages, as they won't be in cache.
        """
```